okhttp/src/main/java/io/grpc/okhttp/OkHttpServerTransport.java

```diff
@@ -381,8 +400,13 @@ public OutboundFlowController.StreamState[] getActiveStreams() {
   void streamClosed(int streamId, boolean flush) {
     synchronized (lock) {
       streams.remove(streamId);
-      if (maxConnectionIdleManager != null && streams.isEmpty()) {
-        maxConnectionIdleManager.onTransportIdle();
+      if (streams.isEmpty()) {
+        if (maxConnectionIdleManager != null) {
+          maxConnectionIdleManager.onTransportIdle();
+        }
+        if (keepAliveEnforcer != null) {
```

Comment:
keepAliveEnforcer won't be null (and ping() assumes it isn't null). Ditto elsewhere